x-pack/test/functional/apps/endpoint/management.ts

```diff
@@ -25,19 +25,61 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
     });
 
     it('displays table data', async () => {
-      const data = await pageObjects.endpoint.getManagementTableData();
-      [
-        'Hostnamecadmann-4.example.com',
-        'PolicyPolicy Name',
-        'Policy StatusPolicy Status',
-        'Alerts0',
-        'Operating Systemwindows 10.0',
-        'IP Address10.192.213.130, 10.70.28.129',
-        'Sensor Versionversion',
-        'Last Activexxxx',
-      ].forEach((cellValue, index) => {
-        expect(data[1][index]).to.equal(cellValue);
-      });
+      const expectedData = [
+        [
+          'Hostname',
+          'Policy',
+          'Policy Status',
+          'Alerts',
+          'Operating System',
+          'IP Address',
+          'Sensor Version',
+          'Last Active',
+        ],
```

Comment:
You might want to comment that this is the column row

Reply:
Good idea.  I will add that on my next PR.